core/templates/pages/release-coordinator-page/release-coordinator-page.component.html

```diff
@@ -0,0 +1,98 @@
+<oppia-release-coordinator-navbar [(activeTab)]="activeTab"></oppia-release-coordinator-navbar>
+
+<div align="center"
+     *ngIf="statusMessage"
+     class="oppia-status-message-container protractor-test-status-message">
+  {{ statusMessage }}
+</div>
+<br>
+<div class="oppia-release-coordinator-page-tabs-container">
+  <oppia-jobs-tab *ngIf="activeTab === TAB_ID_JOBS" (setStatusMessage)="statusMessage=$event"></oppia-jobs-tab>
+
+  <div class="oppia-tlease-coordiator-misc-tab-container" *ngIf="activeTab === TAB_ID_MISC">
+    <div class="oppia-release-coordinator-page-card oppia-page-card oppia-long-text">
+      <h3>Flush the Memory Cache</h3>
+      <div class="oppia-action-buttons-container">
+        <button mat-raised-button color="primary" (click)="flushMemoryCache()">Flush Cache</button>
+        <button mat-raised-button color="primary" (click)="getMemoryCacheProfile()">Get Memory Cache Profile</button>
+      </div>
+      <div class="mt-4" *ngIf="memoryCacheDataFetched">
+        <table class="table text-center view-results-table">
+          <thead>
+            <tr>
+              <td><strong>Total Memory Allocated in Bytes</strong></td>
+              <td><strong>Peak Memory Allocation in Bytes</strong></td>
+              <td><strong>Number of Keys Allocated</strong></td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>{{ memoryCacheProfile.totalAllocatedInBytes }}</td>
+              <td>{{ memoryCacheProfile.peakAllocatedInBytes }}</td>
+              <td>{{ memoryCacheProfile.totalKeysStored }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+
+    <div class="oppia-release-coordinator-page-card oppia-page-card oppia-long-text">
+      <form (ngSubmit)="updatePromoBarConfig()" [formGroup]="promoBarConfigForm">
+        <div class="oppia-promo-bar-config-field">
+          <h3>Promo-bar config</h3>
+          <mat-slide-toggle [color]="'primary'" formControlName="enabled">
+          </mat-slide-toggle>
+        </div>
+        <div class="oppia-promo-bar-config-field row">
+          <label class="col-sm-4 col-form-label">Promo-bar Message:</label>
+          <input matInput class="col-sm-8 form-control display-inline"
+                 type="text"
+                 formControlName="message">
+        </div>
+        <mat-divider></mat-divider>
+        <div class="oppia-action-buttons-container">
+          <button mat-raised-button color="primary" type="submit" [disabled]="!promoBarConfigForm.dirty">Save changes</button>
```

Comment:
```suggestion
          <button mat-raised-button 
                  color="primary" 
                  type="submit" 
                  [disabled]="!promoBarConfigForm.dirty">
            Save changes
          </button>
```

Reply:
This is a comparatively small tag and looks more readable in one line! Anyway, I've fixed it as you have suggested. Done! :)